edx_when/migrations/0005_auto_20190911_1056.py

```diff
@@ -5,23 +5,35 @@
 
 
 class Migration(migrations.Migration):
-
     dependencies = [
         ('edx_when', '0004_datepolicy_rel_date'),
     ]
 
     operations = [
-        migrations.AlterField(
```

Comment:
PostgreSQL doesn't support AlterField?

Reply:
it supports AlterField but for this specific instance casing crash when type casting from interval to double/float.

Reply:
BTW, when you're rebasing, please explain this part in the commit message.